src/AppiumDriver.php

```diff
@@ -609,4 +618,129 @@ public function getStrings($data = [])
     }
+
+    /**
+     * @param $element that accepts a string
+     * @param string to send to $element
+     */
+    public function sendKeys($element, $keys)
+    {
+        $element->setValueImmediate($keys);
+    }
+
+    /**
+     * @return mixed
+     */
+    public function getNetworkConnection()
+    {
+        return $this->driverCommand(static::GET, static::NETWORK);
+    }
+
+    /**
+     * @param array of booleans $data 
+     * ['data'=>true|false , 'wifi'=>true|false, 'airplanemode'=>true|false]
+     *
+     * @return mixed
+     */
+    public function setNetworkConnection($dataArr)
+    {
+	// Note: this does not work due to security exception, 
+	// Android no longer allows this
+
+        $mode = 0;
+        $data = 4;
+        $wifi = 2;
+        $airplane = 1;
+        if (in_array('data', $dataArr) && $dataArr['data'] === true)
+            $mode = $mode | $data;
+        if (in_array('wifi', $dataArr) && $dataArr['wifi'] == true)
+            $mode = $mode | $wifi;
+        if (in_array('airplanemode', $dataArr) && $dataArr['airplanemode'] == true)
+            $mode = $mode | $airplane;
+        return $this->driverCommand(static::POST, static::NETWORK, ['type' => $mode]);
+    }
+
+    /**
+     * @return mixed
+     */
+    public function toggleAirplaneMode()
+    {
+	// Note: this does not work due to security exception, 
+	// Android no longer allows this
+        return $this->driverCommand(static::POST, '/appium/device/toggle_airplane_mode', '');
+    }
+
+    /**
+     * @param startX x-percent at which to start
+     * @param startY y-percent at which to start
+     * @param endX x-percent at which to end
+     * @param endY y-percent at which to end
+     * @param duration (optional) time to take the swipe in ms
+     *
+     * @return mixed
+     */
+    public function swipe($startX, $startY, $endX, $endY, $duration = 800)
+    {
+        $action = $this->initiateTouchAction();
+        $action->press(array('x' => $startX, 'y' => $startY))
+            ->wait($duration)
+            ->moveTo(array('x' => $endX, 'y' => $endY))
+            ->wait($duration)
+            ->release()
+            ->perform();
+        return $this;
+    }
+
+    /**
+     * @return TouchAction
+     */
+    public function initiateTouchAction()
+    {
+        $session = $this->getSession();
+        return new TouchAction($session->getSessionUrl(), $session->getDriver());
+    }
+
+    /**
+     * @return mixed
+     */
+    public function launchApp()
+    {
+        // /appium/app/launch
+        $session = $this->getSession();
+        $url = $this->getSessionUrl()->descend('appium')->descend('app')->descend('launch');
+        $response = $this->getDriver()->curl('POST', $url, null);
+    }
+
+    /**
+     * @param $seconds time during which the application will be put to the
+     * background
+     */
+    public function backgroundApp($seconds)
```

Comment:
in appium function background
https://github.com/tajawal/codeception-appium/blob/ea0736eebbc18a4c3335da5e885b4724d4a6863c/src/Tools/AppiumParser/Output/AppiumCommands.php#L1212